stardog/metadata.csv

```diff
@@ -1,52 +1,507 @@
-metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Are the previous metrics being removed? If so, I would suggest incrementing the version to 2.0.0 rather than 1.0.2 so it's clear that there's a behavior change.